README.md

```diff
@@ -7,6 +7,8 @@ If you have any other questions not listed here, please raise a PR to have it ad
 
 ## Git FAQ
 
+HELEN GORE CHANGE!!
```

Comment:
Helen Gore is fine as she is, she doesn't need change :)

Some of these questions could use some answers however. You can simply push more stuff to your repository and it will reflect in this PR if you want to delete these lines and answer a question :)